server/async-worker/index.js

```diff
@@ -0,0 +1,65 @@
+const { Op } = require("sequelize");
+const callScraper = require("../scraper");
+const { Mention, User } = require("../models");
+
+// A rough draft of how new mentions would get pulled into the db
+// without using an explicit /POST route as well as using
+// a given user's credentials.
+module.exports = async function asyncWorker() {
+  const SECONDS = 1;
+  const MILLISECONDS = 1000;
+  // TODO refactor this into smaller functions
+  console.log("[Scraper] Calling async scraper");
+
+  const users = await User.findAll();
```

Comment:
Since it run every 3 secs, I think this is for the display on main page?
Then I think we only need to scrape every platform selected by the user based on the entered company name, but we don't need to scrape for each user.
Now the logic is more like sending weekly email, but if this is the case, it should run every 7 days instead.

Reply:
Yup the 3 seconds was just for testing purposes, and not much else. I can change it to run weekly now, but I think its useful to test the async worker with a more frequent time period.

This code is intended for the scraping of mentions and sending of weekly emails for what the user enters as their company when signing up.